### PR TITLE
Make getCellIDs's radius configurable

### PR DESCRIPTION
**Pull request by ghost**
✨

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -284,12 +284,13 @@ auth token.
...
 ## `pogobuf.Utils` methods
 ### `getCellIDs(lat, lng)` ⇒ <code>array</code> *(static)*
-Provides cell IDs of nearby cells based on the given coords.
+Provides cell IDs of nearby cells based on the given coords and radius.
 
 | Param | Type | Description |
 | --- | --- | --- |
 | lat | <code>number</code> | Latitude |
 | lng | <code>number</code> | Longitude |
+| radius | <code>number</code> | Radius of the square |
```
Comment (Morgul) [inline README.md:293]: What units are the radius in?

File: pogobuf/pogobuf.utils.js [hunk 2]
```diff
@@ -10,37 +10,36 @@ var s2 = require('s2-geometry').S2;
...
-        cells.push(s2.S2Cell.FromFaceIJ(origin.face, [origin.ij[0] - 2, origin.ij[1] + 1], origin.level).toHilbertQuadkey());
-        cells.push(s2.S2Cell.FromFaceIJ(origin.face, [origin.ij[0] - 2, origin.ij[1] + 2], origin.level).toHilbertQuadkey());
-        cells.push(s2.S2Cell.FromFaceIJ(origin.face, [origin.ij[0] - 1, origin.ij[1] + 2], origin.level).toHilbertQuadkey());
-        cells.push(s2.S2Cell.FromFaceIJ(origin.face, [origin.ij[0], origin.ij[1] + 2], origin.level).toHilbertQuadkey());
-        cells.push(s2.S2Cell.FromFaceIJ(origin.face, [origin.ij[0] + 1, origin.ij[1] + 2], origin.level).toHilbertQuadkey());
-
+         
+        cells.push(origin.toHilbertQuadkey()); // middle block
+         
+        for(var i = 1; i < radius; i++) {
```
Comment (Morgul) [inline pogobuf/pogobuf.utils.js:28]: For those of us who don't understand how the S2 algorithm works, could you provide just a brief (single sentence or two) overview of what you're doing, and why? I see a lot of libraries that take the S2 stuff as 'magic', and then have poor results. It'd be nice if this was just a little better documented so someone unfamiliar could at least follow along a little easier.

File: README.md (hunk 1 above)
Comment (ghost) [inline README.md:293]: The radius is in cells (i.e. Radius of 3 is radius of 3 s2 cells)

Comment (Morgul) [inline README.md:293]: That would be great to mention in the docs, as I'll bet people will assume kilometers.

**Pull request merged**